app/controllers/FakeResponseController.scala

```diff
@@ -28,78 +30,93 @@ import renderer.Renderer
 import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.viewmodels.NunjucksSupport
 
+import scala.concurrent.ExecutionContext
 import scala.xml.Elem
 import scala.xml.{XML => xmlFile}
-import scala.concurrent.ExecutionContext
-import scala.concurrent.Future
 
-class ResponseController @Inject()(
+class FakeResponseController @Inject()(
```

Comment:
@MatthewAlexanderBeck the name FakeResponseController feels quite not right,  how about naming it as NCTSResponseController or something in similar lines

Reply:
@ashwinirudrappa hey Ashwini, do you think EISResponse would also be suitable as that is the end point we are try to mimic?